build/barys

```diff
@@ -22,7 +22,7 @@ BUILD_DIR=build
 REMOVEBUILD=no
 COMPRESS=no
 DEVELOPMENT_IMAGE=no
-SUPERVISOR_TAG=
+SUPERVISOR_VERSION=
```

Comment:
Any specific reasoning for using `VERSION` in barys vs `Release` in jenkins_build.sh? It's all the same to me but it could be aligned?

Reply:
No reason, I'll align them.